File: server/src/schema/budget.ts
export interface Budget {
	year: number,
  
  budgetExpenditureAmount: number,
	budgetIncomeAmount: number,
	expenditureAmount: number,
  incomeAmount: number,
  
	paragraphs: [{
		id: string,
		events: [{
			event: number
			name: string,
			budgetExpenditureAmount: number,
			expenditureAmount: number
		}],
		budgetExpenditureAmount: number,
		expenditureAmount: number
  }],
  
	items: [{
		id: string,
		events: [{
			event: number,
			name: string,
			budgetExpenditureAmount: number,
			budgetIncomeAmount: number,
			expenditureAmount: number,
			incomeAmount: number
		}],
		budgetExpenditureAmount: number,
		budgetIncomeAmount: number,
		expenditureAmount: number,
		incomeAmount: number
	}]
}

File: svg_char_generator/src/pojo/Budget.kt
package cz.cityvizor.svg_char_generator.pojo

import java.math.BigDecimal

data class Etl(val _id: String, val vality: String)

data class Budget(val _id: String, val year: Int, val etl: Etl, val budgetExpenditureAmount: BigDecimal, val expenditureAmount: BigDecimal, val budgetIncomeAmount: BigDecimal, val incomeAmount: BigDecimal)